connector-builder-server/src/main/openapi/openapi.yaml

```diff
@@ -89,25 +91,33 @@ components:
                       type: array
                       description: The RECORD/STATE/LOG AirbyteMessages coming from the read operation for this page
                       items:
-                        $ref: "#/components/schemas/AirbyteProtocol/definitions/AirbyteMessage"
+                        type: object
+                        # $ref: "#/components/schemas/AirbyteProtocol/definitions/AirbyteMessage"
                     request:
                       $ref: "#/components/schemas/HttpRequest"
                     response:
                       $ref: "#/components/schemas/HttpResponse"
     StreamReadRequestBody:
       type: object
       required:
-        - definition
+        - manifest
```

Comment:
There are several changes in this spec to rename "connector definition" to "manifest", following this decision made in slack: https://airbytehq-team.slack.com/archives/C03TP6W8081/p1666214768391589